label_studio_converter/converter.py

```diff
@@ -467,31 +484,11 @@ def convert_to_coco(self, input_data, output_dir, output_image_dir=None, is_dir=
                     logger.warning("Unknown label type: " + str(label))
                     continue
 
-                # get image sizes
-                if first:
-
-                    if 'original_width' not in label or 'original_height' not in label:
-                        logger.warning(f'original_width or original_height not found in {image_path}')
-                        continue
-
-                    width, height = label['original_width'], label['original_height']
-                    image_id = len(images)
-                    images.append({
-                        'width': width,
-                        'height': height,
-                        'id': image_id,
-                        'file_name': image_path
-                    })
-                    first = False
+                if 'original_width' not in label or 'original_height' not in label:
+                    logger.warning(f'original_width or original_height not found in {image_path}')
+                    continue
```

Comment:
As I understand this continue will skip all image that don't have 'original_width' in annotations. However we wanted to include all images from dataset.

Reply:
Fixed